### PR TITLE
useful tag name for MRs, added imageTagPrefix and imageTagSuffix

### PR DESCRIPTION
**Pull request by 123FLO321**
- Useful tag name for MRs: `mrID` (e.g.: `mr1`)
- added `imageTagPrefix` and `imageTagSuffix` options

### CODE REVIEW

**Comment by matootie** [comment]
Interesting addition. It took me a little while to see the purpose in having an image prefix or suffix, when you can just specify the image tag... but now I see it. You can have *GitHub Docker* decide the tag of the image, and prepend or append something you like to it. Nice stuff!

**Comment by 123FLO321** [comment]
@matootie I might even change this up a bit so we can use regex or something in the tag instead.

**Comment by matootie** [comment]
@123FLO321 I very impatiently merged the PR already! 😆

Let's open up a new PR for that, if you'd like.

File: dist/index.js [hunk 1]
```diff
@@ -693,7 +693,18 @@ async function run() {
...
+    if (!imageTag) {
+        const refLast = refArray[refArray.length - 1];
+        if (refLast === "merge" && refArray.length >= 2) {
+            imageTag = "mr" + refArray[refArray.length - 2];
+        } else {
+            imageTag = refLast;
+        }
+    }
+    let imageTagPrefix = core.getInput('imageTagPrefix', { required: false });
+    if (imageTagPrefix) imageName = imageTagPrefix + imageName;
```
Comment (123FLO321) [inline dist/index.js]: I also made a mistake here (and 2 limes below) 
Should be imageTag not imageName

Comment (matootie) [reply dist/index.js]: I've addressed this in #12 

Take a look when you get the chance, and I'll put the fix in a new release.

**Pull request merged**